ipld/src/selector/mod.rs

```diff
@@ -168,12 +168,12 @@ impl SubAssign<u64> for RecursionLimit {
 /// Condition is expresses a predicate with a boolean result.
 ///
 /// Condition clauses are used several places:
-///   - in Matcher, to determine if a node is selected.
-///   - in ExploreRecursive, to halt exploration.
-///   - in ExploreConditional,
+///   - in `Matcher`, to determine if a node is selected.
+///   - in `ExploreRecursive`, to halt exploration.
+///   - in `ExploreConditional`,
 ///
 ///
-/// TODO -- Condition is very skeletal and incomplete.
+/// TO-DO: Condition is very skeletal and incomplete.
```

Comment:
Move this `TODO` out of the documentation.

Reply:
👍